app/assets/stylesheets/scaffolds.scss

```diff
@@ -0,0 +1,65 @@
+body {
```

Comment:
Just curious, how are this css defined? Are they just defaults taken from a library?

Reply:
Yes, I think it's created when running `rails g scaffold`